src/ui/public/filter_manager/__tests__/filter_manager.js

```diff
@@ -114,13 +114,13 @@ describe('Filter Manager', function () {
     checkAddFilters(0, null, 3);
     expect(appState.filters).to.have.length(2);
 
-    let scriptedField = {name: 'scriptedField', scripted: true, script: 1};
+    let scriptedField = {name: 'scriptedField', scripted: true, script: 1, lang: 'painless'};
```

Comment:
Can we add a separate test for the painless scripted field, instead of modifying the existing test? I think we're still supporting Groovy, but recommending painless.